lib/workload/stateless/stacks/bclconvert-interop-qc-pipeline-manager/scripts/trigger_bclconvert_interop_qc.sh

```diff
@@ -21,19 +21,19 @@ WORKFLOW_VERSION="1.3.1--1.21__20240410040204"
 #############
 
 # Inputs
-BCLCONVERT_REPORT_DIRECTORY_URI="icav2://7595e8f2-32d3-4c76-a324-c6a85dae87b5/primary_data/240229_A00130_0288_BH5HM2DSXC/202405302b817f5e/Reports/"
-INTEROP_DIRECTORY_URI="icav2://7595e8f2-32d3-4c76-a324-c6a85dae87b5/primary_data/240229_A00130_0288_BH5HM2DSXC/202405302b817f5e/InterOp/"
-INSTRUMENT_RUN_ID="240229_A00130_0288_BH5HM2DSXC"
+BCLCONVERT_REPORT_DIRECTORY_URI="icav2://ea19a3f5-ec7c-4940-a474-c31cd91dbad4/primary_data/240424_A01052_0193_BH7JMMDRX5/20240606b794561d/Reports/"
+INTEROP_DIRECTORY_URI="icav2://ea19a3f5-ec7c-4940-a474-c31cd91dbad4/primary_data/240424_A01052_0193_BH7JMMDRX5/20240606b794561d/InterOp/"
+INSTRUMENT_RUN_ID="240424_A01052_0193_BH7JMMDRX5"
 
 # RUNTIME
-PROJECT_NAME="trial"
-PROJECT_ID="7595e8f2-32d3-4c76-a324-c6a85dae87b5"
-WORKFLOW_RUN_NAME_PREFIX="umccr__automated__bclconvert_interop_qc__1_3_1__1_21__20240410040204__"
+PROJECT_NAME="development"
+PROJECT_ID="ea19a3f5-ec7c-4940-a474-c31cd91dbad4"
+WORKFLOW_RUN_NAME_PREFIX="umccr__automated__${WORKFLOW_NAME}__1_3_1__1_21__20240410040204__"
 PORTAL_RUN_ID="$(date +%Y%m%d)$(xxd -l 4 -c 4 -p < /dev/random)"
 
 # Outputs
-ANALYSIS_OUTPUT_URI_PREFIX="icav2://7595e8f2-32d3-4c76-a324-c6a85dae87b5/analysis_data/bclconvert_interop_qc/1_3_1--1_21__20240410040204/"
-ICA_LOGS_URI_PREFIX="icav2://7595e8f2-32d3-4c76-a324-c6a85dae87b5/analysis_logs/bclconvert_interop_qc/1_3_1--1_21__20240410040204/"
+ANALYSIS_OUTPUT_URI_PREFIX="icav2://${PROJECT_ID}/analysis_data/${WORKFLOW_NAME}/1_3_1__1_21__20240410040204/"
+ICA_LOGS_URI_PREFIX="icav2://${PROJECT_ID}/analysis_logs/${WORKFLOW_NAME}/${WORKFLOW_VERSION//./_}/"
```

Comment:
Perhaps, these mid paths might have to change accordingly too, Alexis? 
Lines 24, 25, 35, 36.